blockchain/My-Wallet-V3-iOS
# fix(KYC): prevent announcement card from persisting upon logout (IOS-1360)

## Objective

Prevent the *Continue verification* announcement card from persisting upon logging out and logging into or creating a different wallet.

### Additional Changes
- Improve the documentation for the application's settings.
- Add new `clear` and `reset` functions to aid the logout/forget wallet process.
- Fix minor linter issue.

## Description

The value of `shouldShowKYCAnnouncementCard` was not being reset upon logging out.

## How to Test

Begin the KYC process, drop-off and logout of your current wallet. Create a new wallet and observe that the announcement card does not appear.

## Merge Checklist
